src/azure-cli/azure/cli/command_modules/acs/commands.py

```diff
@@ -130,6 +130,7 @@ def load_command_table(self, _):
         g.custom_show_command('show', 'aks_nodepool_snapshot_show',
                               table_transformer=aks_show_nodepool_snapshot_table_format)
         g.custom_command('create', 'aks_nodepool_snapshot_create', supports_no_wait=True)
+        g.custom_command('update', 'aks_nodepool_snapshot_update', supports_no_wait=True)
```

Comment:
CRUD nodepool snapshot are not long running operations, so `supports_no_wait=True` is meanlingless.

Besides, as the model/struct of nodepool snapshot does not have a field named as provisioningState, the wait command will never terminate (as it is to judge whether the resource has reached the expected state by polling and checking the value of the provisioningState field). So I would say you could also remove `g.wait_command('wait')`.